rllib/core/rl_module/rl_module.py

```diff
@@ -192,3 +199,15 @@ def make_distributed(self, dist_config: Mapping[str, Any] = None) -> None:
     @abc.abstractmethod
     def is_distributed(self) -> bool:
         """Returns True if the module is distributed."""
+
+    def as_multi_agent(self) -> "MultiAgentRLModule":
+        """Returns a multi-agent wrapper around this module."""
+        return self.get_multi_agent_class()({"modules": {DEFAULT_POLICY_ID: self}})
```

Comment:
I have a mixed feeling about this.
using cfg as a container to pass internal data may not be a good idea.
why don't we let MARLModule take modules as parameters?
you can then also have one fewer case in ``make_modules()`` above.

Reply:
The usage of this is not clear yet until the trainer_runner is in place, but this is what I had in mind:
```
# inside trainer
module: Union[MARLModule, RLModule] = self.get_module()
marl_module = module.get_multi_agent()
# going forward everything is multi-agent so there should be no more confusion on SA vs. MA
```

Reply:
with this usage in mind what is weird?

Reply:
so this API is completely fine. I just don't know why we need to go the config route, like why not:
```
def as_multi_agent(self) -> "MultiAgentRLModule":
    return self.get_multi_agent_class({DEFAULT_POLICY_ID: self})
```
why wrap yourself which is already a module in a config?
does this makes sense?